docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,59 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database you can use to store private transaction payloads for Orion. Consider using this option as an aid in [disaster recovery](Disaster-Recovery.md).
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+In your database:
+
+1. Create the `store` table by using the provided DDL script, as shown in the following example. Substute your own values for `HOST-NAME`, `USER-NAME`, and `DATABASE-NAME`.
+
+    `psql -h <HOST-NAME> -U <USER-NAME> -d <DATABASE-NAME> -f ~/orion/docs/configuring-orion/database/postgres_storage.sql` 
```

Comment:
For this to work, the user will have to have the Orion source code. So think we should add a step to download Orion source code or clone the Orion repo before running psql.
A thought, perhaps the ddl scripts should be included in the Orion distribution?

Reply:
Forcing to download all Orion source code just ot have the DDL scripts seems a bit overkill...

Reply:
The DDL file will be in the Orion distribution. See https://pegasys1.atlassian.net/browse/OR-326.